cmd/libs/go2idl/openapi-gen/generators/openapi.go

```diff
@@ -40,15 +40,21 @@ const tagOptional = "optional"
 
 // Known values for the tag.
 const (
-	tagValueTrue  = "true"
-	tagValueFalse = "false"
+	tagValueTrue       = "true"
+	tagValueFalse      = "false"
+	tagExtensionPrefix = "x-kubernetes-"
 )
 
-func hasOpenAPITagValue(comments []string, value string) bool {
+func getOpenAPITagValue(comments []string) []string {
 	tagValues := types.ExtractCommentTags("+", comments)[tagName]
 	if tagValues == nil {
-		return false
+		return []string{}
```

Comment:
can this entire function be `return types.ExtractCommentTags("+", comments)[tagName]`?